build/openresty/wasmx/wasmx_repositories.bzl

```diff
@@ -6,18 +6,23 @@ load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("@kong_bindings//:variables.bzl", "KONG_VAR")
 
 def wasmx_repositories():
-    ngx_wasm_module_branch = KONG_VAR["NGX_WASM_MODULE_BRANCH"]
     wasmtime_version = KONG_VAR["WASMTIME_VERSION"]
     wasmer_version = KONG_VAR["WASMER_VERSION"]
     v8_version = KONG_VAR["V8_VERSION"]
     wasmtime_os = KONG_VAR["WASMTIME_OS"]
     wasmer_os = KONG_VAR["WASMER_OS"]
     v8_os = KONG_VAR["V8_OS"]
 
+    ngx_wasm_module_tag = KONG_VAR["NGX_WASM_MODULE"]
+    ngx_wasm_module_branch = KONG_VAR.get("NGX_WASM_MODULE_BRANCH")
+    if ngx_wasm_module_branch:
+        ngx_wasm_module_tag = None
```

Comment:
iirc the `branch` property supports both tag and branch to be checked out (not aligned with what said in docs though), we can simplify this logic as biit